#This msg defines a message for dividing laserscan into 3 separate distances
#It provides mean and minimum values for the three sectors.

time stamp

float64 leftMin
float64 centerMin
float64 rightMin

float64 leftMean
float64 centerMean
float64 rightMean